app/assets/stylesheets/modules/form.scss

```diff
@@ -0,0 +1,20 @@
+@import "variables";
+@import "mixins";
+
+.sign-up {
+  width: 50%;
+  margin-left: auto;
+  margin-right: auto;
+
+  .legend{
+    font-family:$dogstay-font-theme;
+    font-size: 2em;
+  }
+
+  .postfix {
+    width:30%;
```

Comment:
Properties should be ordered background, margin-left, margin-right, width<br>Colon after property should be followed by one space